README.md

```diff
@@ -36,7 +36,7 @@ support package.
 
 ### Quick Start
 
-Jawn supports Scala 2.11, 2.12, and 2.13.0-RC1.
+Jawn supports Scala 2.12, 2.13.0, and dotty.
```

Comment:
Can you change `2.13.0` to `2.13`? The patch version isn't relevant now that 2.13 is out.

Also, it's probably worth clarifying that Jawn isn't currently published for Dotty, just cross-built with it.

Reply:
Also we should capitalize Dotty if we mention it.